packages/fxa-settings/src/pages/Signin/utils.ts

```diff
@@ -66,7 +66,7 @@ export async function handleNavigation(
     // `verified: false`, causing a Sync sign-in issue (see FXA-9837).
     !to?.includes('signin_totp_code')
   ) {
-    firefox.fxaLogin({
+    await firefox.fxaLogin({
```

Comment:
I suspect this is what was causing the intermittant issues, the login event might not have been completely sent before navigating the page.